disk/datadog_checks/disk/disk.py

```diff
@@ -455,3 +466,26 @@ def _compile_tag_re(self):
             except TypeError:
                 self.log.warning('{} is not a valid regular expression and will be ignored'.format(regex_str))
         self._device_tag_re = device_tag_list
+
+    def _get_devices_label(self):
+        """
+        Get every label to create tags
+        """
+        devices_label = {}
+        try:
+            blkid_out, _, _ = get_subprocess_output(['blkid'], self.log)
+            all_devices = [l.strip().split() for l in blkid_out.splitlines()]
```

Comment:
Few things about the parsing logic.

1) `l.strip().split()` this split seems not safe. If you have a space in quote, it won't do what you want.
2) same split/strip could be avoid I think

My suggestion:

```
import re

KEY_VALUE_PATTERN = re.compile(r'(\w+)="(.*?)"') # constant, compile once

input_line = '/dev/sda1: LABEL="MY LABEL" UUID="5eea373d-db36-4ce2-8c71-12ce544e8559" TYPE="ext4"'
device, data = input_line.split(":", 1)

for key, value in re.findall(KEY_VALUE_PATTERN, data):
    print("key: {}, value: {}".format(key, value))

# OUTPUT
# key: LABEL, value:MY LABEL
# key: UUID, value:5eea373d-db36-4ce2-8c71-12ce544e8559
# key: TYPE, value:ext4

```